src/panels/lovelace/header-footer/hui-graph-header-footer.ts

```diff
@@ -27,7 +28,8 @@ export class HuiGraphHeaderFooter extends LitElement
 
   @property() public hass?: HomeAssistant;
   @property() protected _config?: GraphHeaderFooterConfig;
-  @property() private _coordinates?: any;
+  @property() private _coordinates?: number[][];
+  @property() private _loadingInitialData: boolean = true;
```

Comment:
Does this need to be a property? When coordinates changes it should update right?

It may not request an update if its set to undefined though. Not 100%. But you may not need this to be a property.

Reply:
Good point. Made a non-property